### DISCUSSION TITLE
Lesson 1: Deploying to a “live” network

### DISCUSSION THREAD

**Discussion by propeltrain** [smartcontractkit/full-blockchain-solidity-course-py#1795] [Q&A] [answered]
I’m having multiple issues:
1. On MetaMask, the Rinkeby test network is not found, so how do I get test eth?
2. The *Environment* on Remix does not have *Injected Web3* like in the tutorial.

Now, how do I solve the Issue with test eth and *Injected Web3* Environment to deploy the whole thing to a “live” network?

PS: I’m a beginner to coding, so this question might be very basic.

**Comment by cromewar** [comment, marked as answer]
Hello @trainpropel 

> On MetaMask, the Rinkeby test network is not found, so how do I get test eth?

Rinkeby network was deprecated, use goerli instead, you can get test eth here: https://faucets.chain.link/

> The Environment on Remix does not have Injected Web3 like in the tutorial. 

Select the new option "Injected Provider" is exactly the same, they just changed the name.

> PS: I’m a beginner to coding, so this question might be very basic.

Make any questions as you like we will try to provide support for all of them.

**Reply by propeltrain** [reply]
Thanks for your reply, but I still have the issues:

a. When I used this https://faucets.chain.link/, I got 10 LINK, but I don’t know how to use them.
![AB6B6B7C-DDCA-4A56-8C28-39F6770E5E2E](https://user-images.githubusercontent.com/109322953/205443789-5789b73c-d806-4383-998c-4a292e76353e.png)

b. Then I used https://goerli-faucet.mudit.blog/, but faucet shows that I have insufficient fund for gas fees.
     
![F9AB2FD1-9E62-4D37-B3EB-9077CDEC231E](https://user-images.githubusercontent.com/109322953/205443936-746dd150-770e-4567-8fe4-3a26bae22a42.png)

c. Then I went to alchemy.com for Goerli ETH, but no luck
  
![E969395B-6A7B-4C61-B923-231EFA28B0C9](https://user-images.githubusercontent.com/109322953/205444091-664b5546-819e-4b94-9d6b-f587f845465d.png)

d. So, I still have no Goerli ETH 
![036E1FF6-CE97-47E0-8024-0F27A24FDD57](https://user-images.githubusercontent.com/109322953/205444139-7747d530-9638-44c7-be67-4078ed6e83c8.png)


e. I am using Brave browser, and my wallet address:  0xeefDD40FD42c70050F02dC2F9c4CA866D2828137

**Reply by cromewar** [reply]
I just sent you over some eth:  https://goerli.etherscan.io/tx/0x87fde2babfd8627b91f03031e2c7b4f62771cb641c82bbe159e43370d7a46881 can you check please?

1. seems chainlink removed the goerli faucet, so you can try with: https://goerlifaucet.com/
2. try using a different account on alchemy.

**Reply by propeltrain** [reply]
Thank you soo verryyy much for sending the eth. :) I was looking for some Goerli eth for a while. Highly appreciate it. :)

![075F9115-D76C-463F-AFBC-CA1DD392DCCA](https://user-images.githubusercontent.com/109322953/205485560-001971c8-a7eb-4f8b-a9f2-58b0d5c63f5f.png)

**Reply by cromewar** [reply]
that should be enough for your fist transactions.

**Reply by propeltrain** [reply]
Thank you once again. All the issues have been solved. :)

**Reply by cromewar** [reply]
Feel Free to make any other question.

**Reply by propeltrain** [reply]
Sure. I certainly will. Thanks
